artifactory-users-report/Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+@Library('pipeline-library@pull/348/head') _
+
+pipeline {
+	triggers {
+    // Execute once per hour
+		cron('@hourly')
+	}
+	options {
+		// This pipeline takes 1-2 minutes max to execute
+		timeout(time: 10, unit: 'MINUTES')
+    lock(resource: 'infra-reports-artifactory-users', inversePrecedence: true)
```

Comment:
```suggestion
```
No need to lock this job, it wasn't in the previous version and it doesn't take too much time to execute.

Reply:
It wasn't on the previous job, but having PRs and shorter time between jobs, sonds safer (hence the addon)

Reply:
OK for the longer jobs, this one could do without it since it's quite quick though

Reply:
The lock ensures that 2 builds cannot be run concurently. Since the report is updated with overwrite as the end, it must be enforced.